data/aadii0408.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Aaditya Champaneri || Aadi ",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  {
```

Comment:
Please follow the Read me for the Json file. tags dont need to be in a separate block

Reply:
1. https://linkfree.eddiehub.io/docs/quickstart
2. https://github.com/EddieHubCommunity/LinkFree/blob/main/data/eddiejaoude.json

Link 1 will give you a visual demo.
Link 2 will show you the format of how the json has to be.